.github/workflows/publish-artifacts.yml

```diff
@@ -86,7 +86,9 @@ jobs:
         cd oras
         cp ../target/${{ env.RUST_TARGET }}/release/attestation-agent .
         tar cJf attestation-agent.tar.xz attestation-agent
-        arch_tag="${{ github.sha }}-${{ matrix.platform.tee }}_${{ matrix.platform.arch }}"
+        arch="${{ matrix.platform.arch }}"
```

Comment:
Can we add a comment here to tell why we do this? I thin currently it's clear, but in later time we could forget that

Reply:
Yes, adding a comment will be helpful.